src/main/kotlin/step2/calculator/Inputs.kt

```diff
@@ -0,0 +1,104 @@
+package step2.calculator
+
+import step2.calculator.vo.CalculatorInput
+import step2.calculator.vo.Number
+import step2.calculator.vo.Operation
+
+class Inputs(
+    private val numbers: List<Number>,
+    private val operations: List<Operation>,
+) {
+    private var numbersPosition: Int = 0
+    private var operationsPosition: Int = 0
+
+    val hasNextNumber: Boolean
+        get() {
+            return numbers.size > numbersPosition
+        }
```

Comment:
```suggestion
    val hasNextNumber: Boolean
        get() = numbers.size > numbersPosition
```

이렇게 하면 조금 더 깔끔해질 것 같아요!